site/app/templates/submission/grade_inquiry/Discussion.twig

```diff
@@ -125,6 +125,9 @@
     </div>
 
     <script>
+        window.onload = function() {
+            saveDraft();
+        };      
```

Comment:
Shouldn't this load the draft instead of saving it?

Reply:
Hi William, I changed the function name "saveDraft" to "loadDraft"